server/src/app.ts

```diff
@@ -14,11 +14,17 @@ app.use(express.static("public"));
 app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 
+// ---- Routers ----
+
 app.get("/", (req: Request, res: Response) => {
   const spaFilePath = path.resolve(__dirname, "..", "public");
   res.sendFile(spaFilePath);
 });
 
+import { default as experienceRouter } from './routes/experiences';
```

Comment:
nit: imports go at the top -- I'll add some prettier and eslint config to automatically catch this